unstructured/partition/pdf.py

```diff
@@ -22,6 +21,8 @@
 from pdfminer.pdfpage import PDFPage
 from pdfminer.pdftypes import PDFObjRef
 from pdfminer.utils import open_filename
+from PIL import Image as PILImage
+from unstructured_inference.inference.layout import DocumentLayout
```

Comment:
This needs to be imported from within the function it's used, or we're picking this up as a base level dependency, which we don't want to do.